### PR TITLE
Added new guide - cd with gke

### PR DESCRIPTION
**Pull request by felicianotech**
This adds a new guide on how to deploy to Google Container Engine using CircleCI and Google Container Registry.

Related to:
- https://github.com/circleci/docker-hello-google/issues/2
- https://github.com/circleci/docker-hello-google/pulls

### CODE REVIEW

File: jekyll/_docs/continuous-deployment-with-google-container-engine.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+provides their own registry to store your images with direct support for gcloud, 
+GKE and K8s.
+
+## Setup
+
+### Prerequisites
+
+This guide makes the following assumptions:
+
+1. Your project source-code is hosted on a CircleCI compatible repository.
```
Comment (kimh) [inline jekyll/_docs/continuous-deployment-with-google-container-engine.md]: I think it's either `source code` or just `code`.

**Comment by kimh** [comment]
LGTM.  It would be nice if you put a link to green build of https://circleci.com/gh/circleci/docker-hello-google

File: jekyll/_docs/continuous-deployment-with-google-container-engine.md (hunk 1 above)
Comment (felicianotech) [inline jekyll/_docs/continuous-deployment-with-google-container-engine.md]: :+1:

**Comment by felicianotech** [comment]
Waiting for Kevin or someone to review the PR in the example project repo.

**Comment by felicianotech** [comment]
@kimh  Added a link to a build per your suggestion. Thanks! :smile:

**Pull request merged**